### ISSUE TITLE
"Workshop", "Contribute" and "Github" link is not visible on applying resize: A11y_Maow_Home Page_Resize.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/moaw#126]
### GitHub Tags
#A11yTCS; #A11ySev2; #WCAG1.4.4; #Win11; #GH_Maow_Web_Mar24; #DesktopWeb; #Maow; #AILimited; #Resize Text; #A11yMAS;

### Environment Details:
Application Name: Maow
#URL: https://moaw.dev/
Operating system: Windows 11
Enterprise Insider Preview 22H2 (OS build 25324.1000)
Microsoft Edge: Version 118.0.2048.1 (Official build) dev (64-bit)

**Prerequisites:** 
Change the display resolution to 1280*720 and change browser zoom levels up to 200%. 
### Repro Steps:

1. Hit the URL https://moaw.dev/
2. Moaw screen will get open.
3. TAB till Github link.
4. Verify that "Workshop", "Contribute" and "Github" link is visible on applying resize or not.

### Actual:
"Workshop", "Contribute" and "Github" link is not visible on applying resize.

### Expected:
"Workshop", "Contribute" and "Github" link should be visible on applying resize.

### User Impact:
Low visibility users will face difficulty to access the links present in the screen.
### Attachment:
![Bug_126](https://github.com/microsoft/moaw/assets/124244478/a6b72bb8-f173-4016-9131-14bdb08f9599)

**Comment by sinedied** [comment]
@AnKushSingh05 this one has been fixed, could you validate it?

**Comment by AnKushSingh05** [comment]
Verified the issue in below environment
https://moaw.dev/
Issue is fixed. Hence closing the bug

https://github.com/microsoft/moaw/assets/124244478/aa234be5-82d8-4e54-a7d5-128feea66deb
